definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -257,9 +257,9 @@
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespondentSolicitorName",
+    "CaseFieldID": "CoRespondentSolicitorAddress",
```

Comment:
In my story, I had taken the liberty of putting the name first as I thought it was more intuitive.
I had shown it to Jeremy.
Can you please mention it to him that you're inverting the order? I don't mind either way. Whatever is best for the user.

Reply:
Oow yeah I had noticed this wasn't sure whether it was intentional. Will let him know yep.